thorlcr/mfilemanager/thmfilemanager.cpp

```diff
@@ -518,9 +518,12 @@ class CFileManager : public CSimpleInterface, implements IThorFileManager
                 ForEachItemIn(gn, groupNames)
                 {
 #ifdef _CONTAINERIZED
-                    Owned<IStoragePlane> plane = getStoragePlane(groupNames.item(gn));
-                    assertex(plane);
-                    curDir.append(plane->queryPrefix());
+                    if (!globals->getPropBool("@_dafsStorage"))
+                    {
+                        Owned<IStoragePlane> plane = getStoragePlane(groupNames.item(gn));
+                        assertex(plane);
+                        curDir.append(plane->queryPrefix());
```

Comment:
@jakesmith  Doesn't this mean curDir won't be set?

Reply:
@ghalliday - yes if _dafsStorage is set, then curDir is blank, so it's same as if storage plane has no prefix.
It means that 'dir' will be left blank, and makePhysicalPartName will use the default (no diroverride).